src/osx/carbon/font.cpp

```diff
@@ -146,6 +146,34 @@ class WXDLLEXPORT wxFontRefData: public wxGDIRefData
 
     wxFontEncoding GetEncoding() const { return m_info.GetEncoding(); }
     
+    void SetLetterSpacing( wxDouble spacing )
+    {
+        //if ( m_info.m_spacing != spacing )
+        //{
+        //    m_info.SetLetterSpacing( spacing );
+        //    Free();
+        //}
+
+        // Note: do NOT set kCTKernAttribute to zero, that's used to disable kerning! (tristate attr)
+        if (!spacing)
+        {
+            return;
+        }
+        CFMutableDictionaryRef dict = reinterpret_cast<CFMutableDictionaryRef>(m_ctFontAttributes);

```

Comment:
Same thing here, seems we first create a CFMutableDictionary which we store in a CFDictionary member (a superclass?), but we need the mutable dictionary to SetValue.

Reply:
If we're going to modify its values, then it seems natural to just make `m_ctFontAttributes` a mutable dictionary in the first place, otherwise it's just confusing.